src/middlewared/middlewared/api/v25_10_0/virt_instance.py

```diff
@@ -20,6 +19,8 @@
 ]
 
 
+# Some popular OS choices
+OS_ENUM = Literal['DEBIAN', 'FEDORA', 'UBUNTU', 'WINDOWS', 'ARCHLINUX', None]
```

Comment:
Are these defined in Incus somewhere? If we don't have to make any OS-specific tweaks I'd be inclined to just have something like `Literal['LINUX', 'WINDOWS', 'FREEBSD', None]`